### PR TITLE
SEAR-75 Sort search results based on facility name

### PR DESCRIPTION
**Pull request by sunilb0575**
#### Which User story does this relate to (link)?
-https://osi-cwds.atlassian.net/browse/SEAR-75

#### Brief feature description
-Sort facility search results by facility name. 

#### Does this PR block/effect other stories?
-No

#### Do any commands needs to be run in order to run the project with this PR?
-No

#### Code coverage percent diff as a result of this PR?
-No

### CODE REVIEW

**Comment by Nasir2712** [comment]
if you can update the ```query_builder_spec.rb``` file to include sort by name tests, that would be helpful, If not you can just update the current tests with name instead of ``` _score```.

**Comment by sunilb0575** [comment]
@Nasir2712 Changed two of the tests to test sort by 'name', left one as is, to test sort by '_score'.

**Comment by Nasir2712** [comment]
any chance you can rebase?

**Comment by sunilb0575** [comment]
@Nasir2712 already did, want to keep that second commit, as it's addressing change in test case.

**Comment by Nasir2712** [comment]
ya, it is a part of this story so it should be rebased.

**Comment by sunilb0575** [comment]
ok

File: app/javascript/sagas/fetchSearchResultsSaga.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {SEARCH_RESULTS_FETCH, fetchSuccess, fetchFailure} from 'actions/searchAc
 // worker saga: makes the api call when watcher saga sees the action
 export function * fetchSearchResults (action) {
   try {
-    const url = '/facilities/search' + '?from=' + action.payload.urlParams.fromValue + '&size=' + action.payload.urlParams.sizeValue
+    const url = '/facilities/search' + '?from=' + action.payload.urlParams.fromValue + '&size=' + action.payload.urlParams.sizeValue + '&sort=' + action.payload.urlParams.sort + '&order=' + action.payload.urlParams.order
```
Comment (ratneshraval) [inline app/javascript/sagas/fetchSearchResultsSaga.js]: Maybe declare `const = action.payload` so you don't have to keep chaining everywhere in the method.

**Pull request merged**